vivisect/qt/funcgraph.py

```diff
@@ -501,7 +501,7 @@ def _renderMemory(self):
 
             try:
                 addr = self.vw.parseExpression(expr)
-            except Exception, e:
+            except Exception as e:
```

Comment:
This arose out of me trying to view the code fragments in funcgraph. That failed, but at least we got some cleanup out of it.